operators/tidb-operator/1.3.0/tidb-operator.v1.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,608 @@
+
+    TiDB ("Ti" stands for Titanium) is an open-source, distributed, NewSQL
+    database that supports Hybrid Transactional and Analytical Processing (HTAP)
+    workloads. It is MySQL compatible and features horizontal scalability,
+    strong consistency, and high availability. TiDB can be deployed on-premise
+    or in-cloud.
+
+    ## About this Operator
+
+
+    TiDB Operator manages TiDB clusters on Kubernetes and automates tasks
+    related to operating a TiDB cluster. It makes TiDB a truly cloud-native
+    database.
+
+
+    ## Features
+
+
+    * Safely scaling the TiDB cluster
+
+
+    TiDB Operator empowers TiDB with horizontal scalability on the cloud.
+
+
+    * Rolling update of the TiDB cluster
+
+
+    Gracefully perform rolling updates for the TiDB cluster in order, achieving
+    zero-downtime of the TiDB cluster.
+
+
+    * Multi-tenant support
+
+
+    Users can deploy and manage multiple TiDB clusters on a single Kubernetes
+    cluster easily.
+
+
+    * Automatic failover
+
+
+    TiDB Operator automatically performs failover for your TiDB cluster when
+    node failures occur.
+
+
+    * Kubernetes package manager support
+
+
+    By embracing Kubernetes package manager Helm, users can easily deploy TiDB
+    clusters with only one command.
+
+
+    * Automatically monitoring TiDB cluster at creating
+
+
+    Automatically deploy Prometheus, Grafana for TiDB cluster monitoring.
+
+
+    ## Quick Start
+
+    You can follow our [Get
+    Started](https://docs.pingcap.com/tidb-in-kubernetes/stable/get-started)
+    guide to quickly start a testing Kubernetes cluster and play with TiDB
+    Operator on your own machine.
+
+
+    ## Documentation
+
+
+    You can see our documentation at PingCAP website for more in-depth
+    installation and instructions for production:
+
+
+    - [English](https://docs.pingcap.com/tidb-in-kubernetes/stable)
+
+    - [简体中文](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable)
+
+
+    All the TiDB Operator documentation is maintained in the [docs-tidb-operator
+    repository](https://github.com/pingcap/docs-tidb-operator).
+
+
+    ## Community
+
+
+    Feel free to reach out if you have any questions. The maintainers of this
+    project are reachable via:
+
+
+    - [TiDB Community Slack](https://pingcap.com/tidbslack/) in the
+    [#sig-k8s](https://tidbcommunity.slack.com/archives/CHD0HA3LZ) channel
+
+    - [Filing an issue](https://github.com/pingcap/tidb-operator/issue) against
+    this repo
+
+
+    Pull Requests are welcome! Check the [issue
+    tracker](https://github.com/pingcap/tidb-operator/issue) for
+    `status/help-wanted` issues if you're unsure where to start.
+
+
+    If you're planning a new feature, please file an issue or join
+    [#sig-k8s](https://tidbcommunity.slack.com/archives/CHD0HA3LZ) channel to
+    discuss first.
+
+    ## Prerequisites for enabling this Operator
+  maturity: stable
+  version: 1.3.0
+  replaces: tidb-operator.1.1.6
```

Comment:
@KanShiori there should be `tidb-operator.v1.1.6` to match the `name` from `1.1.6`. You have older versions without `v` and newer with `v`.

Reply:
Fixed, thanks a lot!